static/app/utils/api/apiOptions.spec.tsx

```diff
@@ -22,44 +21,41 @@ const wrapper = ({children}: {children?: React.ReactNode}) => (
 );
 
 describe('apiOptions', () => {
-  afterEach(() => {
-    MockApiClient.clearMockResponses();
-  });
-  test('should encode path parameters correctly', () => {
+  it('should encode path parameters correctly', () => {
     const options = apiOptions.as<unknown>()(
-      '/projects/$orgSlug/$projectSlug/releases/$releaseVersion/',
```

Comment:
had to update all the tests so they're using real routes

i tried to pick stuff that's near the top of the list, but had the properties the test cares about.

Reply:
the `/projects` one also exists, just the variables are named differently:

```
'/projects/$organizationIdOrSlug/$projectIdOrSlug/releases/$version/'
```